Source/JavaScriptCore/offlineasm/x86.rb

```diff
@@ -594,7 +547,7 @@ def x86Operands(*kinds)
         result = []
         kinds.size.times {
             | idx |
-            i = isIntelSyntax ? (kinds.size - idx - 1) : idx
+            i = idx
```

Comment:
Seems like we should remove this line and rename the parameter.

Reply:
Done